Assets/Scripts/MovementControllers/PillPlayerController.cs

```diff
@@ -0,0 +1,71 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[RequireComponent(typeof(Rigidbody))]
+public class PillPlayerController : MonoBehaviour
+{
+    public Planet attractor;
+    public Camera firstPersonCamera;
+    public float movementSpeed;
+    [SerializeField] private float maxSpeed = 0;
+
+    private Rigidbody body;
+    // Start is called before the first frame update
+    void Start()
+    {
+        body = GetComponent<Rigidbody>();
+        //A bit of a hack to give the player a starting planet
+        attractor = GameObject.Find("Sun").transform.parent.transform.GetChild(1).GetComponent<Planet>();
+        Vector3 directionNearestPlanet = attractor.transform.position - transform.position;
+        Physics.Raycast(transform.position, directionNearestPlanet, out RaycastHit hit);
+        transform.SetParent(attractor.transform);
+
+        //Put the player above the ground
+        transform.position = hit.point - (directionNearestPlanet.normalized) * 5;
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        HandleInput();
+        Gravity.KeepUpright(transform, attractor.transform);
+        Gravity.Attract(transform.position, body, attractor.transform.position, attractor.mass);
+    }
+
+    private void HandleInput()
+    {
+        Vector3 movementVector = Vector3.zero;
+        if (Input.GetKey(KeyCode.W))
```

Comment:
You don't really need to seperate the different inputs like this. This can be done in one line, refer to this link:
https://docs.unity3d.com/ScriptReference/CharacterController.Move.html